static/app/views/organizationIntegrations/integrationRow.tsx

```diff
@@ -96,7 +96,7 @@ const IntegrationRow = (props: Props) => {
   return (
     <PanelRow noPadding data-test-id={slug}>
       <FlexContainer>
-        <PluginIcon sentryApp={sentryApp} isColor size={36} pluginId={slug} />
+        {customIcon ? customIcon : <PluginIcon size={36} pluginId={slug} />}
```

Comment:
@leeandher You can use `customIcon ?? <PluginIcon size={36} pluginId={slug} />`

Reply:
nice